app/create/controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = function ($scope, Postcard) {
+
+  $scope.payload = { mail_type: 'usps_first_class' };
+  $scope.loading = false;
+  $scope.error = null;
+  $scope.postcard = null;
+
+  $scope.create = function (payload) {
+    $scope.loading = true;
+    $scope.error = null;
```

Comment:
do we need this line and the next? idk haha

Reply:
Yes. On every new request, it needs to cleared out.
[https://www.notion.so/lob/Controllers-46fa970c00ae481f8bbb1a37d5dfc03c#ce532bd037524fcda86e4cf3430dffa6]